### DISCUSSION TITLE
Heavy coil whine on monitor refresh

### DISCUSSION THREAD

**Discussion by 00x0A** [ChefKissInc/NootRX#38] [Q&A] [answered]
### macOS Version

Ventura/Sonoma (Only Navi 21)

### What is your CPU's model?

Intel i5 12400F

### What is your GPU's model?

AMD Radeon RX 6700 XT

### Please describe the behaviour in detail.

When high speed movement such as scrolling occurs on the screen the GPU makes a coil whine noise that's not present in Linux or Windows 10/11. I have tried 60hz, 120hz and 165hz which all causes the GPU to make a coil whine noise, could perhaps be something with the driver or power management being handled. I've tried setting PP,PP_WorkLoadPolicyMask to 1 without any success

### What should've happened instead?

GPU should coil whine but only under heavy load i.e not for scrolling.

### Bug report submission terms

- [X] I am using the latest version of NootRX.
- [X] I am on the latest minor and patch version of macOS (ex. 7 is the minor and 10 is the patch of macOS 11.7.10).
- [X] The bug only happens with NootRX enabled.
- [X] I have ruled out any external factors that could interfere, including, but not limited to, using OCLP or WhateverGreen.

**Comment by VisualEhrmanntraut** [comment]
Uhm uhm uhm, am I gonna be making GPUs explode with this kext??

**Comment by VisualEhrmanntraut** [comment]
> I've tried setting PP,PP_WorkLoadPolicyMask to 1 without any success

@00x0A that would be FULLSCREEN3D_WORKLOAD, did you mean POWERSAVING_WORKLOAD which is 2?

**Comment by VisualEhrmanntraut** [comment]
(coil whine = high power usage so make GPU use less power :-))

**Comment by VisualEhrmanntraut** [comment]
Oh wait, did you use the DeviceProperties method? Because NootRX does not allow you to set custom properties like WhateverGreen; you have to edit the Info.plist of the kext.

**Comment by 00x0A** [comment]
> Oh wait, did you use the DeviceProperties method? Because NootRX does not allow you to set custom properties like WhateverGreen; you have to edit the Info.plist of the kext.

Youre a chill dev, just wanted to thank you for the work thus far bro. I'm gonna check the info.plist of the kext and whether or not i can make it stop haha. 

It might have something to do with apple's amd drivers since windows 11 also has a heavy coil whine until you install the proper drivers, Linux does the have this issue out of the box

**Comment by VisualEhrmanntraut** [comment]
Did you resolve it

**Comment by 00x0A** [comment]
Yes, changing the value in info.plist to 1 fixed my issue

**Reply by june5910aaaa** [reply]
I am a newbie and I encountered the same problem as you. Could you teach me how to fix it by changing the value in info.plist to 1?
Many thanks～

**Comment by VisualEhrmanntraut** [comment, marked as answer]
For future readers: This issue does not exist in the latest version.
